### DISCUSSION TITLE
rds.ModifyDBClusterInput (ScalingConfiguration.MinCapacity) Fail

### DISCUSSION THREAD

**Discussion by ptilotta** [aws/aws-sdk-go-v2#1314] [General]
Hi,

We use this function in a AWS Lambda, but we're having the error of

runtime error: invalid memory address or nil pointer dereference: errorString
[{"path":"github.com/aws/aws-lambda-go@v1.24.0/lambda/errors.go","line":39,"label":"lambdaPanicResponse"},{"path":"github.com/aws/aws-lambda-go@v1.24.0/lambda/function.go","line":36,"label":"(Function).Invoke.func1"},{"path":"runtime/panic.go","line":679,"label":"gopanic"},{"path":"runtime/panic.go","line":199,"label":"panicmem"},{"path":"runtime/signal_unix.go","line":394,"label":"sigpanic"},{"path":"Lamda/main.go","line":55,"label":"HandlerAcus"},{"path":"reflect/value.go","line":460,"label":"Value.call"},{"path":"reflect/value.go","line":321,"label":"Value.Call"},{"path":"github.com/aws/aws-lambda-go@v1.24.0/lambda/handler.go","line":124,"label":"NewHandler.func1"},{"path":"github.com/aws/aws-lambda-go@v1.24.0/lambda/handler.go","line":24,"label":"lambdaHandler.Invoke"},{"path":"github.com/aws/aws-lambda-go@v1.24.0/lambda/function.go","line":64,"label":"(Function).Invoke"},{"path":"reflect/value.go","line":460,"label":"Value.call"},{"path":"reflect/value.go","line":321,"label":"Value.Call"},{"path":"net/rpc/server.go","line":377,"label":"(*service).call"},{"path":"runtime/asm_amd64.s","line":1357,"label":"goexit"}]

the code is

`func main() {
	lambda.Start(proceso)
}

func proceso(parametros Parametros) {
	cfg, err := config.LoadDefaultConfig(context.TODO(), config.WithRegion("us-east-1"))
	if err != nil {
		log.Fatalf("unable to load SDK config, %v", err)
	}

	acusN32 := int32(parametros.Acus)

	svc := rds.NewFromConfig(cfg)

	// Configura el objeto rds.ModifyCurrentDBClusterCapacityInput
	var param rds.ModifyDBClusterInput
	param.ScalingConfiguration.MinCapacity = &acusN32
	param.DBClusterIdentifier = &parametros.Instancia

	/* resp, err := svc.ModifyCurrentDBClusterCapacity(context.TODO(), &param) */
	resp, err := svc.ModifyDBCluster(context.TODO(),  &param)

	if err != nil {
		log.Fatalf("falló al modificar ACUS, %v", err)
	} else {
		log.Println("Los ACUS de " + *&parametros.Instancia + " quedaron seteados en " + string(*resp.DBCluster.Capacity))
	}
}
`

The line error is this 
`	var param rds.ModifyDBClusterInput
	param.ScalingConfiguration.MinCapacity = &acusN32
	param.DBClusterIdentifier = &parametros.Instancia`

What's Wrong ?

Thanks

**Comment by marc-ostrow** [comment]
What version were you targeting.  The latest service operation API for ModifyCurrentDBClusterCapacity (as of 17a3908) does not have a `ScalingConfiguration` attribute in its input structure, nor does it have the `DBCluster` attribute in its output structure.
